hadoop-plugin/src/main/groovy/com/linkedin/gradle/hadoopdsl/checker/RequiredParametersChecker.groovy

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2015 LinkedIn Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.linkedin.gradle.hadoopdsl.checker
+
+import com.linkedin.gradle.hadoopdsl.BaseStaticChecker
+import com.linkedin.gradle.hadoopdsl.Workflow
+import com.linkedin.gradle.hadoopdsl.job.Job
+import org.gradle.api.Project
```

Comment:
Can you move _com.linkedin.gradle.hadoopdsl.job.Job_ to the beginning of imports and give a new line before importing _org.gradle.api.Project_ ?